packages/tracing/src/browser/request.ts

```diff
@@ -107,8 +107,6 @@ type PolymorphicRequestHeaders =
 export const defaultRequestInstrumentationOptions: RequestInstrumentationOptions = {
   traceFetch: true,
   traceXHR: true,
-  tracingOrigins: DEFAULT_TRACING_ORIGINS,
```

Comment:
L: is `DEFAULT_TRACING_ORIGINS` used anywhere else now that this is gone? Can we get rid of it?

Reply:
[3dd0cce](https://github.com/getsentry/sentry-javascript/pull/6238/commits/3dd0cce62a1df8b385b7613d2497c868102b60c8)